src/content/protocols/uniswap-v3.md

```diff
@@ -0,0 +1,131 @@
+| ProxyAdmin                       | upgradeAndCall      | triggers the upgrade of the NonFungibleTokenPositionDescriptor contract which allows to change the token descriptions and then call a function in the new contract.                                                                       | TimeLocked DAO contract (GovernerBravo) |
+| TransparentUpgradeableProxy      | changeAdmin         | updates the proxy's admin: the account with the rights to upgrade the proxy's implementation.                                                                                                                                             | ProxyAdmin                              |
+| TransparentUpgradeableProxy      | upgradeTo           | upgrades the NonFungibleTokenPositionDescriptor contract which allows to change the token descriptions.                                                                                                                                   | ProxyAdmin                              |
+| TransparentUpgradeableProxy      | upgradeToAndCall    | upgrades the NonFungibleTokenPositionDescriptor contract which allows to change the token descriptions and then call a function in the new contract.                                                                                      | ProxyAdmin                              |
+| TimeLock                         | queueTransaction    | queues a transaction that can be executed once a delay (between 2 and 30 days) has passed. This can impact the own TimeLock's settings (change admin, set delays) or interaction with any other contract the TimeLock has permissions on. | GovernorBravoDelegator                  |
+| TimeLock                         | cancelTransaction   | cancels a pending transaction.                                                                                                                                                                                                            | GovernorBravo                           |
+| TimeLock                         | executeTransaction  | executes a transaction that was previously queued, if the corresponding delay has passed.                                                                                                                                                 | GovernorBravoDelegator                  |
+| GovernorBravoDelegator           | \_setImplementation | updates the implementation of the GovernorBravo (DAO) contract. Can only be triggered by the DAO itself.                                                                                                                                  | TimeLocked DAO contract (GovernerBravo) |
```

Comment:
Looks good, but can you enrich the impact text a bit, so that it's more clear.

What I little framework I suggest:
1st sentence: technically what it does: Allows the owner to set a fee percentage that is deducted from the LPs fees.
2nd sentence: what is means inside of the protocol: It only affects the pool where the function is called. 
3rd sentence: state how it could be abused, or how it is protected to be abused: The fee is required to be less than 10% of the total accumulated fees. It only affects future accumulated fees.

sometimes 1st sentence is enough, but most of the time 2 sentences are the best.

Reply:
here are some rows which I think need some clarification

Reply:
> | ProxyAdmin                       | renounceOwnership   | abandons ownership of the contract, making the implementation immutable.    

What cannot be changed anymore?

Reply:
> | UniswapV3Factory                 | setOwner            | changes the owner to a new address. 

The DAO can appoint a new owner which can set fees on various pools (setProtocolFee), collect fees on behalf of the protocol and allow new tick spaces for new deployed pools.

like that, I think the idea should be that reading a row itself should give enough clarification where you want to read next. Otherwise the sentence leaves you bit in the dark, depending on the context and protocol

Reply:
> | UniswapV3Factory                 | enableFeeAmount     | enables a fee amount for a given tick spacing.    

It also allows new tick spaces. And fees and tick spaces are limited and required to meet certain boundaries, please incorporate that

Reply:
> | UniswapV3Factory (UniswapV3Pool) | collectProtocol     | withdraw accumulated protocol fees to a custom address. 

full sentence, starting with capital letter (this counts for all rows)

Reply:
> | ProxyAdmin                       | renounceOwnership   | abandons ownership of the contract, making the implementation immutable.    

ie. the NonFungibleTokenPositionDescriptor implementation cannot be changed via the Proxy contract anymore